pyproject.toml

```diff
@@ -31,6 +31,7 @@ metrics = [
     "ptflops>=0.7",
 ]
 visual = ["ipykernel", "graphviz", "torchview"]
+examples = ["napari","jupyterlab","jupyter","jupytext","nbconvert"]
```

Comment:
`jupyter` is a metapackage which includes some of the other things:

> For more efficient installation of what you need, all Jupyter components installed by pip install jupyter can be installed separately. For example:
notebook - Jupyter Notebook
jupyterlab - JupyterLab (added to metapackage v1.1)
ipython - IPython (terminal)
ipykernel - IPython Kernel for Jupyter
jupyter-console - terminal Jupyter client
nbconvert - convert notebooks between formats
ipywidgets - interactive widgets package for IPython

Reply:
https://pypi.org/project/jupyter/

Reply:
```suggestion
examples = ["napari", "jupyter", "jupytext"]
```

Reply:
gotcha. thanks!